### PR TITLE
Fixes race condition with admin proc calls and load admins

### PR DESCRIPTION
**Pull request by silicons**
this might end badly

### CODE REVIEW

File: code/modules/admin/admin_ranks.dm [hunk 1]
```diff
@@ -346,7 +349,7 @@ GLOBAL_PROTECT(admin_ranks)
 				D = new(R,adm_ckey)	//new admin
 
 			var/client/C = GLOB.directory[adm_ckey]	//find the client with the specified ckey (if they are logged in)
-			D.associate(C)						//link up with the client and add verbs
+			INVOKE_ASYNC(D, .proc/associate, C)						//link up with the client and add verbs
```
Comment (kevinz000) [inline code/modules/admin/admin_ranks.dm]: i'm assuming by this point permissions have already been checked and i'm also assuming invoke_async won't pass usr into a proc?

**Comment by Cyberboss** [comment]
How about you read the code for IsAdminAdvancedProcCall first https://github.com/tgstation/tgstation/blob/master/code/modules/admin/verbs/debug.dm#L128

**Comment by Cyberboss** [comment]
Change it

**Comment by kevinz000** [comment]
Kk give me a bit I'll be on something I can edit in in like a few hours kkthz

**Comment by kevinz000** [comment]
Thx*

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Change it

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**